### DISCUSSION TITLE
🚨 The Solidity compiler in Remix is only supported from solidity version 0.8.26 onwards.

### DISCUSSION THREAD

**Discussion by ricardomazuera** [Cyfrin/foundry-full-course-cu#2437] [General]
In [Lesson 4](https://updraft.cyfrin.io/courses/solidity/simple-storage/create-solidity-smart-contract) in [Solidity Smart Contract Development](https://updraft.cyfrin.io/courses/solidity) @PatrickAlphaC says that in the course we will use solidity version 0.8.19; however, Remix now only supports from 0.08.26 onwards.

![Captura de Pantalla 2024-09-09 a la(s) 7 43 04 p m](https://github.com/user-attachments/assets/d5b98b7c-f03d-46ab-8bcb-5359e9cfbd47)

**Comment by ricardomazuera** [comment]
Sorry about that. It was my mistake when I was testing in the compiler.

**Reply by EngrPips** [reply]
It's OK, Keep crushing it.
